mem-store.js

```diff
@@ -39,6 +39,8 @@ class MemStore {
     }
 
     const data = this._store.get(cid)
+
+    // TODO: Change this to refs
```

Comment:
I'd like to do this soon, probably while I'm refactoring and updating the `ipfs-log` tests

Reply:
Leaving for now, one refactor at a time